product/roundhouse.console/Properties/AssemblyInfo.cs

```diff
@@ -4,4 +4,4 @@
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 
 [assembly: Guid("a7a8d35d-cb38-4f12-a388-41c0fd21bf20")]
-[assembly: AssemblyKeyFile("..\\..\\RoundhousE.snk")]
\ No newline at end of file
+//[assembly: AssemblyKeyFile("..\\..\\RoundhousE.snk")]
```

Comment:
Signing is less important nowadays to most folks in the community. The question is whether organizations (the quiet unheard group) still feel the same way. I like the concept of that signer you mentioned.

Reply:
I can look into it. It basically signes all unsigned dlls in the project. But we'll need to use it in all the projects, then, bercause the DLL changes (it's basically a different one, as the Assembly qualified name changes).

Reply:
👍